Podcast/UserDetailViewController.swift

```diff
@@ -0,0 +1,413 @@
+                headerView.frame = headerFrame
+                profileTableView.tableHeaderView = headerView
+            }
+        }
+    }
+    
+    override func stylizeNavBar() {
+        navigationController?.navigationBar.tintColor = .offWhite
+        navigationController?.navigationBar.setBackgroundImage(UIImage(), for: .default)
+        navigationController?.navigationBar.shadowImage = UIImage()
+        navigationController?.navigationBar.backgroundColor = .clear
+    }
+    
+    override func mainScrollViewSetup() {
+        mainScrollView?.contentInsetAdjustmentBehavior = .never
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        UIApplication.shared.statusBarStyle = .lightContent
+        stylizeNavBar()
+        if (navigationController?.view != navBar.superview) {
+            navigationController?.view.insertSubview(navBar, belowSubview: (navigationController?.navigationBar)!)
+            navBar.snp.makeConstraints { make in
+                make.top.leading.trailing.equalToSuperview()
+                make.height.equalTo(UIApplication.shared.statusBarFrame.height + 44)
+            }
+        }
+    }
+    
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        fetchAll()
+    }
+    
+    override func viewWillDisappear(_ animated: Bool) {
+        super.viewWillDisappear(animated)
+        UIApplication.shared.statusBarStyle = .default
+        navigationController?.navigationBar.setBackgroundImage(nil, for: .default)
+        navigationController?.navigationBar.shadowImage = nil
+        navBar.removeFromSuperview()
+    }
+    
+    func fetchAll() {
+        fetchRecasts()
+        fetchSubscriptions()
+    }
+    
+    func fetchRecasts() {
+        let recastsRequest = FetchUserRecommendationsEndpointRequest(userID: user.id)
+        recastsRequest.success = { (endpointRequest: EndpointRequest) in
+            guard let results = endpointRequest.processedResponseValue as? [Episode] else { return }
+            self.recasts = results
+            self.isLoading = false
+            self.userDetailHeaderView.setSubscriptions(hidden: self.isLoading)
+            self.profileTableView.reloadData()
+        }
+        recastsRequest.failure = { (endpointRequest: EndpointRequest) in
+            // Display error
+            print("Could not load user favorites, request failed")
+            self.isLoading = false
+            self.userDetailHeaderView.setSubscriptions(hidden: self.isLoading)
+        }
+        System.endpointRequestQueue.addOperation(recastsRequest)
+    }
+    
+    func fetchSubscriptions() {
+        let userSubscriptionEndpointRequest = FetchUserSubscriptionsEndpointRequest(userID: user.id)
+        userSubscriptionEndpointRequest.success = { (endpointRequest: EndpointRequest) in
+            guard let subscriptions = endpointRequest.processedResponseValue as? [Series] else { return }
```

Comment:
I see # of Subscribers being a better sorting metric here since its most likely people looking at your profile